docs/user_guides/fs/feature_group/create_external.md

```diff
@@ -39,7 +39,7 @@ To create an external feature group using the HSFS APIs you need to provide an e
         GROUP BY ss_store_sk, sales_date
     """
 
-    fg = feature_store.create_on_demand_feature_group(name="sales",
+    fg = feature_store.create_external_feature_group(name="sales",
```

Comment:
change api to external instead of on-demand

Reply:
thanks, good catch